subsys/bluetooth/controller/ll_sw/ull_scan.c

```diff
@@ -622,41 +728,116 @@ static void ticker_cb(uint32_t ticks_at_expire, uint32_t remainder,
 }
 
 #if defined(CONFIG_BT_CTLR_ADV_EXT)
-void ull_scan_done(struct node_rx_event_done *done)
+static uint8_t is_scan_update(uint8_t handle, uint16_t duration,
+			      uint16_t period, struct ll_scan_set **scan,
+			      struct node_rx_pdu **node_rx_scan_term)
+{
+	*scan = ull_scan_set_get(handle);
+	*node_rx_scan_term = (void *)(*scan)->node_rx_scan_term;
```

Comment:
Missing NULL check for `*scan`?

Reply:
The caller uses a constant valid handle value of `SCAN_HANDLE_1M` or `SCAN_HANDLE_PHY_CODED`